runsc/boot/config.go

```diff
@@ -112,6 +113,20 @@ func MakeWatchdogAction(s string) (watchdog.Action, error) {
 	}
 }
 
+// MakeRefsLeakMode converts type from string
+func MakeRefsLeakMode(s string) (refs.LeakMode, error) {
+	switch strings.ToLower(s) {
+	case "nocheck":
+		return refs.NoLeakChecking, nil
+	case "warning":
+		return refs.LeaksLogWarning, nil
+	case "traces":
```

Comment:
You could remove this option, given that it's not much different than logging a warning.

Reply:
This is incorrect. The traces mode logs full stack traces from the allocation site instead of just the name of the type that was leaked. This is extremely useful for debugging. Arguably the mode which isn't terribly useful is the warning mode as it gives you very little to go on for resolving the issue.